Functions.Templates/Templates/EventGridTrigger-Python/metadata.json

```diff
@@ -4,7 +4,8 @@
     "name": "Event Grid trigger",
     "language": "Python",
     "category": [
-        "$temp_category_experimental"

```

Comment:
Python is an experimental language though (not GA), so shouldn't this remain experimental?

Reply:
Actually none of the python templates have the category. We shouldn't be using category "experimental" at all in V2.

Reply:
Ah, ok.